README.md

```diff
@@ -1,10 +1,48 @@
-# Dashboard
+# Kubeflow Dashboard
 
-This repository is being migrated from kubeflow/kubeflow. Fell free to help us moving it according to the Option 3 in https://github.com/kubeflow/kubeflow/issues/7549.
+[Kubeflow Dashboard](https://www.kubeflow.org/docs/components/central-dash/overview/) is the web-based hub of a Kubeflow Platform.
+It exposes the access controlled web interfaces for Kubeflow components and more.
 
-- We need to set up the new kubeflow/dashboard repository readme, owners file, and issue templates.
-- We should merge (or close) any obvious PRs for the dashboard components, and write down links for any we aren't ready for (so we don't loose track after migration)
-- The commit PR (#xxx) suffix in the new repository should be rewritten as (kubeflow/kubeflow#xxx) so we don't break the links
-- We should only move the folders under components/ that actually correspond to the new dashboard components (using git-filter-repo)
-- We should only migrate the necessary GitHub actions
+> ⚠️ __Note__ ⚠️
+> 
+> We are currently moving the Kubeflow Dashboard codebase from [`kubeflow/kubeflow`](https://github.com/kubeflow/kubeflow) to this repository ([`kubeflow/dashboard`](https://github.com/kubeflow/dashboard)).
+> Please see [`kubeflow/kubeflow#7549`](https://github.com/kubeflow/kubeflow/issues/7549) for more information.
 
+## What is Kubeflow Dashboard?
+
+Key features of Kubeflow Dashboard include:
+
+- Access to the [web interfaces](https://www.kubeflow.org/docs/components/central-dash/overview/#navigation) of Kubeflow components.
+- Authorization using [Kubeflow Profiles](https://www.kubeflow.org/docs/components/central-dash/profiles/) and Kubernetes Namespaces.
+   - _Note, authentication depends on your Kubeflow Distribution, and is not directly handled by Kubeflow Dashboard._
```

Comment:
Here we should link to the Dex and oauth2 proxy documentation in kubeflow/manifests

Reply:
I think those links would be brittle, and end up with 404 issues when we inevitably move things in the manifests repo.

But also, not all distributions will use those tools, so it might confuse users, hence why the current PR just says it "depends on your Kubeflow Distribution".

Reply:
but kubeflow/manifests is not a distribution. So rather say "it depends on how you installed Kubeflow" and link to https://www.kubeflow.org/docs/started/installing-kubeflow/

Reply:
```suggestion
   - _Note, authentication depends on how you [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/), and is not directly handled by Kubeflow Dashboard, but oauth2-proxy and or DEX._
```

Reply:
Made similar changes in https://github.com/kubeflow/dashboard/pull/75/commits/a48f1ca9de3c2146c2f8158d98b567b63592726b